libs/libarchfpga/src/physical_types.h

```diff
@@ -840,6 +853,10 @@ struct t_logical_block_type {
     std::vector<t_physical_tile_type_ptr> equivalent_tiles; ///>List of physical tiles at which one could
                                                             ///>place this type of netlist block.
 
+    std::unordered_map<int, const t_pb_graph_pin*> pb_pin_num_map; /* pb_pin_num_map[pin logical number] -> pb_graph_pin ptr} */
```

Comment:
pb_pin_logical_num_to_pb_graph_pin_mapping  

may be better name?

Reply:
I changed it to pin_logical_num_to_pb_pin_mapping. If you think of any other name please let me know